### ISSUE TITLE
Version numbers don't match

### ISSUE THREAD

**Issue by cassidyjames** [endlessm/endless-key-flatpak#69]
We have three places version numbers are shown:
- **0.3** Flathub app listing/App Center (from AppStream MetaInfo)
- **0.16.0b5 (0.1)** GTK About window (from Meson?)
- **Manatee v6.47.0** In-page About page

Can we plan to align these? Otherwise it's going to be hard to know what version someone is actually using.

**Comment by dylanmccall** [comment]
Oh dammit, I just realized our last few releases didn't actually increment the version in meson.build. We really need to add a pre-commit check for that.

**Comment by starnight** [comment]
> Oh dammit, I just realized our last few releases didn't actually increment the version in meson.build. We really need to add a pre-commit check for that.

Oops!  Sorry, I forgot the one in `meson.build`.  My fault.

And, back to the version number of endless-key-flatpak:
* Current version is temporary.  Because, it had not included kolibri-explore-plugin before #26, but it has/merged now. :)
* [kolibri-explore-plugin](https://github.com/endlessm/kolibri-explore-plugin/releases)'s version string has 3 numbers.  But, only uses first two numbers (major & minor) actually.
* After the welcome-screen is merged, I think it is good to sync the version with kolibri-explore-plugin.  For examples from:
  -  [Windows](https://github.com/endlessm/endless-key-app/tags): the first two numbers are the same of **kolibri-explore-plugin's version**.  The 3rd number is build number.  Windows APPX/MISX defines 4th number as 0.  ex. `6.47.1.0`
  - [Chrome OS](https://github.com/endlessm/kolibri-installer-android/blob/8294cd0a63878b2f4ee986a555f36e8a6e9e7dd4/scripts/version.py#L98-L106): the first two numbers are the same of **kolibri-explore-plugin's version**.  The 3rd number is incremented build number. ex. `Manatee 6.47-402`.
  
So, I suggest using the same scheme as Windows and Chrome OS: Sync the version's first two numbers of kolibri-explore-plugin.  And, the 3rd number is the build number.

**Comment by dylanmccall** [comment]
I made https://github.com/endlessm/endless-key-flatpak/pull/74 as quick and easy solution here. The _smart_ thing to do is probably to adopt `bump2version` here, like we do for projects like [kolibri-explore-plugin](https://github.com/endlessm/kolibri-explore-plugin). We can get it to update a `.version` file and get the root meson.build to read from there.

**Comment by dbnicholson** [comment]
I don't think we should do tie the endless-key-flatpak version to the kolibri-explore-plugin version. What do we call the version when we make changes beyond kolibri-explore-plugin releases? It also essentially means we need to make an endless-key-flatpak release every time we make a kolibri-explore-plugin release. Otherwise the versions will be mismatched again. Which then ripples through to the org.endlessos.Key flathub build. We should be able to make new org.endlessos.Key releases when the explore releases but not endless-key-flatpak.

I think it would be better if we had a way to specify the app version to the explore plugin so it could show that in addition to (or instead of) the plugin version.
